components/brave_new_tab_ui/containers/newTab/index.tsx

```diff
@@ -648,14 +694,38 @@ class NewTabPage extends React.Component<Props, State> {
         imageSrc={this.imageSource}
         imageHasLoaded={this.state.backgroundHasLoaded}
         colorForBackground={colorForBackground}
+        hasSponsoredRichMediaBackground={hasSponsoredRichMediaBackground}
         data-show-news-prompt={((this.state.backgroundHasLoaded || colorForBackground) && this.state.isPromptingBraveNews && !defaultState.featureFlagBraveNewsFeedV2Enabled) ? true : undefined}>
         <OverrideReadabilityColor override={ this.shouldOverrideReadabilityColor(this.props.newTabData) } />
         <BraveNewsContextProvider>
         <EngineContextProvider>
+
+        {
+          hasSponsoredRichMediaBackground && this.sponsoredRichMediaBackgroundInfo &&
```

Comment:
Nit: looks like `hasSponsoredRichMediaBackground` could be eliminated from this condition (since it's true iif `this.sponsoredRichMediaBackgroundInfo`).